packages/cli/src/services/frontend.service.ts

```diff
@@ -160,6 +160,9 @@ export class FrontendService {
 			},
 			onboardingCallPromptEnabled: config.getEnv('onboardingCallPrompt.enabled'),
 			executionMode: config.getEnv('executions.mode'),
+			concurrency: {
```

Comment:
IMO this, and any other frontend changes for the paywall should be a separate PR. We should first ship a way to just set a concurrency limit, and then make sure that it works without any issues for a couple of weeks before we start using it to show the paywall.

Reply:
The plan for cloud is to set this limit according to the user's cloud plan, while self-hosters can freely set a limit or not. Beta period for testing should apply as usual. Let me run this by Product.